src/UhdmAst.cpp

```diff
@@ -3220,20 +3245,25 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
         });
         return assert;
     }
-    case vpiUnsupportedStmt:
-        v3info("\t! This statement is unsupported in UHDM: " << file_name << ":" << currentLine);
+    case vpiUnsupportedStmt: {
+        auto* fl = make_fileline(obj_h);
+        fl->v3info("\t! This statement is unsupported in UHDM: " << file_name << ":" << currentLine);
         // Dummy statement to keep parsing
         return new AstTime(make_fileline(obj_h),
```

Comment:
In other places we throw an error. If we print only information, it is possible that we won't notice it.

Reply:
Fixed.